plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergTrinoRestCatalogConnectorSmokeTest.java

```diff
@@ -66,7 +64,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     }
 
     @Override
-    protected QueryRunner createQueryRunner()
+    protected DistributedQueryRunner createIcebergQueryRunner()
```

Comment:
after https://github.com/trinodb/trino/pull/17498/commits/aa16b41bd2b032128f83b48674ccf0fdafdf0d59#r1211695896 you should be able to revert those renames